plugins/modules/rudder_server_settings.py

```diff
@@ -128,74 +127,108 @@ def _send_request(self, url, data=None, headers=None, method="GET"):
                 ),
             )
         elif status_code == 403:
-            self._module.fail_json(failed=True, msg="Permission Denied")
+            self._module.fail_json(failed=True, msg='Permission Denied')
```

Comment:
I think it is better to let the last case error handler handle this, people will know that a 403 is a permissions issue, but they will not know that "Permission Denied" is from the API.

Reply:
fixup !